Symfony/CS/Fixer/PSR0/Psr0Fixer.php

```diff
@@ -152,7 +152,14 @@ public function supports(\SplFileInfo $file)
 
         $filenameParts = explode('.', $file->getBasename(), 2);
 
-        if (!isset($filenameParts[1]) || 'php' !== $filenameParts[1]) {
+        if (
+            // ignore file without name
+            '' === $filenameParts[0]
+            // ignore file with extension other than php
+            || (!isset($filenameParts[1]) || 'php' !== $filenameParts[1])
+            // ignore file started by digit (as class cannot be named that way)
+            || is_numeric($filenameParts[0][0])
```

Comment:
Maybe `ctype_digit` would be better, since `is_numeric` will return true for hexadecimals for PHP 5 and not for PHP 7?